### ISSUE TITLE
Ident rule improperly flagging block comments

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#3046]
ESLint version 1.0.0-rc-1.

I have code that looks like this:

```js
/*
 * When you first setup a webhook, GitHub sends a ping to test the server.
 * That ping has a X-GitHub-Event header set to "ping" with a dummy payload.
 * For pings, we should respond with a 204 to let GitHub know we"re okay.
 */
case "ping":
    res.status(204).end();
    break;
```

The `indent` rule is flagging each line after the opening comment marker as having the wrong indentation. Output:

```
  58:13  error  Expected indentation of 8 characters   indent
  59:17  error  Expected indentation of 12 characters  indent
  60:17  error  Expected indentation of 12 characters  indent
```

It seems like multiline comments need to have an exception in this case.

**Comment by gyandeeps** [comment]
Does your setting have indent case set to true?
Can you provide the more of the switch statement?

**Comment by nzakas** [comment]
Strange, I'll double-check this later.

**Comment by gyandeeps** [comment]
@nzakas Is this still an issue?

**Comment by nzakas** [comment]
I am. I can't reproduce in the online demo, though I'm using 1.0.0-rc-1 and the demo isn't.

**Comment by nzakas** [comment]
Oops, I meant yes this is still an issue for me. Can't quite figure out what's going on.

**Comment by gyandeeps** [comment]
whats your config for the rule?

**Comment by nzakas** [comment]
Haha, caught me. :)

```
indent: [2, 4]
```

**Comment by gyandeeps** [comment]
I cant reproduce it using the same config. Are you sure you dint have `tabs` instead of `spaces`?
Also can you share code where it can be reproduced?

**Comment by nzakas** [comment]
I can't share the code, otherwise I would have. :(

Yes, I'm sure I don't have tabs.

**Comment by BYK** [comment]
I can't repro either.

**Comment by gyandeeps** [comment]
Will close now. Feel free to reopen if the issue persists.
